design.md

```diff
@@ -1,13 +1,37 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+The target for the design is users who want to program Picobot and have either a basic knowledge of 
+programming or logic flow.
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+We converted Picobot's implicit conditionals into explicit 'if' statements which programmers will be familiar
+with and non-programmers may find closer to natural language and thus more readable. We also made the movememnts
+more explicit by forcing the user to type 'move' and a direction when specifying a move. We think this will
+better distinguish an action taken by the robot from a conditional check of the robot's surroundings. We created funtion names such as 'isEmpty(Direction)' and 'isWall(Direction)' to clarify how the robot checks its surroundings.
+Our overall design principle is valuing code clarity and readability over brevity so the language is more 
+accessible to novice programmers. We also chose to include task which set an initial state and allow the user
+to group rules together. We think this will allow users to write more modular code by grouping related behaviors.  
 
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
+We think it's easier to express the difference between a bot's move and a conditional check of its surroundings. 
+In the original code they both simply used N, E, W, S letters for both and the only distinguishing factor was the
+placement of those letters in the code. In our code the syntax is different for each behavior. 
 
 ## What behaviors are more difficult to express in your design than in Picobot’s original design? If there are no such behaviors, why not?
+Our syntax somewhat abstracts the idea of states and state machines so it could be less intuitive for an 
+expert in that area. However, we think that even for that type of user having string names for states instead of
+integers will make the code more readable. We did trade brevity for clarity so the code will be much longer 
+and more repetitive for a given program so a Picobot expert would spend more time coding in our syntax than the 
+original syntax.
 
```

Comment:
I'd say additionally that it takes a little bit longer to specify complex surroundings, since you need to write "isWall" or "isEmpty" in front of the directions, as well as "and" between them. I do like your way better than the default picobot, but it does seem like a short program would be at least quicker to write in picobot.